scripts/bump.sh

```diff
@@ -15,7 +15,6 @@ fi
 RELEASE_CHANNEL=$1
 echo "RELEASE_CHANNEL: $RELEASE_CHANNEL"
 
-PRISMA_CHANNEL=$RELEASE_CHANNEL
```

Comment:
huh?

Reply:
oh - that wasn't intentional. Will fix this.